helm/charts/determined/values.yaml

```diff
@@ -128,3 +128,6 @@ telemetry:
 
 ## A user friendly name to identify this cluster by.
 # clusterName: Dev
+
+# This is the default password for the admin and determined accounts
```

Comment:
Helm comments usually follow the convention of putting the variable name first so something like this:
```
defaultPassword sets the password for the admin and determined user accounts. 
```